conda-store-server/conda_store_server/server/app.py

```diff
@@ -179,6 +179,8 @@ def initialize(self, *args, **kwargs):
 
         self.conda_store = CondaStore(parent=self, log=self.log)
 
+        self.conda_store.ensure_directories()
```

Comment:
Note for other reviewers: this runs:
```py
os.makedirs(self.store_directory, exist_ok=True)
```
because store_directory might not exist.